ansible/roles/cartodb.postgresql/tasks/main.yml

```diff
@@ -0,0 +1,55 @@
+---
+
+ - name: Add cartodb postgresql apt repository
+   apt_repository: repo=ppa:cartodb/postgresql-9.3 update_cache=yes
+
+ - name: Install postgresql client packages
+   apt: name={{ item }} state=present
+   with_items:
+    - libpq5
+    - libpq-dev
+    - postgresql-client-{{ postgresql_version }}
+    - postgresql-client-common
+
+ - name: Install postgresql server packages
+   apt: name={{ item }} state=present
+   with_items:
+    - postgresql-{{ postgresql_version }}
+    - postgresql-contrib-{{ postgresql_version }}
+    - postgresql-server-dev-{{ postgresql_version }}
+    - postgresql-plpython-{{ postgresql_version }}
+
+ - name: Add CartoDB pg-schema-trigger repo
+   apt_repository: repo=ppa:cartodb/pg-schema-trigger state=present update_cache=yes
+
+ - name: Install postgresql schema triggers
+   apt: name=postgresql-9.3-pg-schema-triggers state=present
+
+ - name: Configure PostgreSQL host-based authentication
+   template: src=pg_hba.conf.j2 dest=/etc/postgresql/{{ postgresql_version }}/main/pg_hba.conf
+   notify:
+     - Restart PostgreSQL
+
+ - name: Add cartodb publicuser
+   command: createuser publicuser --no-createrole --no-createdb --no-superuser
+   sudo: True
+   sudo_user: postgres
+   # TODO: Come up with better way to handle user already created
```

Comment:
There's a `postgresql_user` module -- would that work? http://docs.ansible.com/ansible/postgresql_user_module.html